cmd/syncthing/gui.go

```diff
@@ -1017,6 +1020,30 @@ func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request) {
 	sendJSON(w, evs)
 }
 
+func (s *apiService) getDiskEvents(w http.ResponseWriter, r *http.Request) {
```

Comment:
Heh, I was wondering if you'd want that or not.  I gambled that you wouldn't, so what just an extra bool parameter then in the input to differentiate?